pandas/core/generic.py

```diff
@@ -2423,22 +2423,23 @@ def copy(self, deep=True):
         data = self._data.copy(deep=deep)
         return self._constructor(data).__finalize__(self)
 
-    def convert_objects(self, convert_dates=True, convert_numeric=False,
-                        convert_timedeltas=True, copy=True):
+    def convert_objects(self, datetime=True, numeric=False,
+                        timedelta=True, coerce=False, copy=True):
         """
         Attempt to infer better dtype for object columns
 
         Parameters
         ----------
-        convert_dates : boolean, default True
-            If True, convert to date where possible. If 'coerce', force
-            conversion, with unconvertible values becoming NaT.
-        convert_numeric : boolean, default False
+        datetime : boolean, default True
+            If True, convert to date where possible.
+        numeric : boolean, default False
```

Comment:
need to use ``deprecate`` decorator (on the old kw's). These will show a warning and then get reassigned to the new kws. So it will still work.